### PR TITLE
Enable utility feature

### PR DESCRIPTION
**Pull request by othmanalobayyat**
This is the initial implementation of the utility feature based on the requirements.

### CODE REVIEW

File: util/config.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+config1:abc
+
+config2:123
```
Comment (othmanm7mad) [inline util/config.txt]: This config value should be 567. 
123 was the old value.

File: util/feature.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+feature 1 implementation
+
+ feature 2 implementation
```
Comment (othmanm7mad) [inline util/feature.txt]: "Let's add a bit more to the feature 2 implementation. I think it should handle multiple widgits.

**Review by othmanm7mad — CHANGES_REQUESTED** [review]
Please fix the config values and add the implementation to feature2

File: util/feature.txt (hunk 2 above)
Comment (othmanalobayyat) [reply util/feature.txt]: @othmanm7mad see the latest commit for the change

**Review by othmanalobayyat — COMMENTED** [review]
Modified code based on review

**Review by othmanm7mad — APPROVED** [review]

**Pull request merged**